Implementation/FindMyBLEDevice/FindMyBLEDevice/ViewModels/AboutViewModel.cs

```diff
@@ -10,9 +12,15 @@ public class AboutViewModel : BaseViewModel
         public AboutViewModel()
         {
             Title = "About";
-            OpenWebCommand = new Command(async () => await Browser.OpenAsync("https://aka.ms/xamarin-quickstart"));
+            
+            OpenMapPageCommand = new Command(async () => await Shell.Current.GoToAsync($"{nameof(MapPage)}"));
+            OpenStrengthPageCommand = new Command(async () => await Shell.Current.GoToAsync($"{nameof(StrengthPage)}"));
```

Comment:
Opening a new stack to show the burger menu instead of a back button in the pages would make more sense imo.
Was this a conscious decision?

I think this can be archieved by prefixing the routes with "//", but don't quote me on that.
@Chippelius should be able to help you with that.

Reply:
I agree. 
And yes, simply putting // in between the opening quotes and the opening curly bracket will do the trick.

Reply:
Done